api/models.py

```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```

Comment:
Your template code doesn't change significantly if you use text codes instead of number codes. I'd also suggest leaving the model as it is for now.